### PR TITLE
Bug 1601281 addon_names query

### PR DESCRIPTION
**Pull request by jklukas**
As originally proposed in https://bugzilla.mozilla.org/show_bug.cgi?id=1601281#c2

### CODE REVIEW

File: sql/telemetry_derived/addon_names_v1/query.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
+WITH grouped AS (
+  SELECT
+    addons.key AS addon_id,
+    addons.value.name AS addon_name,
+    COUNT(*) AS occurrences
```
Comment (relud) [inline sql/telemetry_derived/addon_names_v1/query.sql]: This should probably be:
```suggestion
    COUNT(DISTINCT client_id) AS occurrences
```

File: sql/telemetry/addons_daily/view.sql [hunk 2]
```diff
@@ -4,4 +4,4 @@ AS
 SELECT
   *
 FROM
-  `moz-fx-data-shared-prod.telemetry_derived.addons_daily_v1`
\ No newline at end of file
+  `moz-fx-data-shared-prod.telemetry_derived.addons_daily_v1`
```
Comment (jklukas) [inline sql/telemetry/addons_daily/view.sql]: Unrelated change that was picked up by the formatter.
